packages/core/src/TextSplitter.ts

```diff
@@ -55,10 +53,9 @@ export function cjkSentenceTokenizer(sentence: string): string[] {
   return slist.filter((s) => s.length > 0);
 }
 
-export const defaultParagraphSeparator = EOL + EOL + EOL;
```

Comment:
wasn't the problem that we're not using `export const defaultParagraphSeparator = EOL + EOL`?